source/postprocess/visualization.cc

```diff
@@ -161,6 +161,38 @@ namespace aspect
           }
       };
 
+      /**
+       * This Postprocessor will generate the output variables of velocity,
+       * pressure, temperature, and compositional fields on the surface of the
+       * domain.
+       */
+      template <int dim>
+      class SurfaceBaseVariablePostprocessor: public VisualizationPostprocessors::SurfaceOnlyVisualization<dim>, public BaseVariablePostprocessor< dim >
```

Comment:
Could you reword the description to just say something like "adapt the previous class for the case of surface output; the only difference is that it prepends the string 'surface_' to each output variable name.